src/Compilers/CSharp/Portable/Symbols/TypeSymbolExtensions.cs

```diff
@@ -803,6 +804,18 @@ public static bool ContainsDynamic(this TypeSymbol type)
         private static readonly Func<TypeSymbol, object, bool, bool> s_containsDynamicPredicate = (type, unused1, unused2) => type.TypeKind == TypeKind.Dynamic;
 
         /// <summary>
+        /// Return true if the type contains any tuples.
+        /// </summary>
+        internal static bool ContainsTuple(this TypeSymbol type) =>
```

Comment:
If it is a tuple, but does not have named fields, do we still return true here?
I wonder if we want to serialize  `(int, int) Foo()` any different than `ValueTuple<int, int> Foo()` ?